src/audio_buffer.cpp

```diff
@@ -31,7 +31,9 @@ void AudioBuffer::destroy() {
 }
 
 bool AudioBuffer::checkForDiscontinuity(int threshold) const {
-    for (int i = 0; i < m_bufferSize - 1; ++i) {
+    int i = 0;
+    int m_bufferSizeMinusOne = m_bufferSize - 1;
```

Comment:
This may be a minor optimization when debugging, but most modern compilers will do this optimization for us. I don't think there's a benefit unless the termination condition includes calling a function.